src/math/src/keccak256.cairo

```diff
@@ -0,0 +1,65 @@
+use keccak::{cairo_keccak};
+
+#[generate_trait]
+impl U64Impl of U64Trait {
+    /// Converts a little-endian byte slice to a 64-bit unsigned integer
+    ///
+    /// # Arguments
+    ///
+    /// * `self` - A `Span<u8>` slice of size n <=8.
```

Comment:
Should this function return a result then if the expected size should be `<= 8`?